libs/ledger-live-common/src/deviceSDK/commands/quitApp.ts

```diff
@@ -0,0 +1,16 @@
+import { Observable } from "rxjs";
+import Transport from "@ledgerhq/hw-transport";
+
+
+export function quitApp(
```

Comment:
Looks like there is some prettier/indent issue ;)

Reply:
done